src/tests/test_method.py

```diff
@@ -304,6 +304,10 @@ def testValueParamsArgs(self):
         self.assertTrue(result[1] == 2)
         self.assertTrue(result[2] == 3)        
 
+    def testNonParamsArrayInLastPlace(self):
```

Comment:
makes sense - no more questions! the answer as to why both are not allowed
is given here: http://stackoverflow.com/a/7580295/2230844

On Wed, Apr 6, 2016 at 11:29 AM, omnicognate <notifications@github.com>
wrote:

> In src/tests/test_method.py
> <https://github.com/pythonnet/pythonnet/pull/201#discussion_r58737494>:
>
> > @@ -304,6 +304,10 @@ def testValueParamsArgs(self):
> >          self.assertTrue(result[1] == 2)
> >          self.assertTrue(result[2] == 3)
> >
> > +    def testNonParamsArrayInLastPlace(self):
>
> I just tried it, but C# won't let me have both
>
> TestNonParamsArrayInLastPlace(int i1, params int[] ints)
>
> and
>
> TestNonParamsArrayInLastPlace(int i1, int[] ints)
>
> It says they have the same name and parameter types, error CS0111.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/pythonnet/pythonnet/pull/201/files/4117162e2e98a0d2458c067ac7ea4b69e9a5d8e2#r58737494>
>